src/Temporal/WorkerOptionsFactory.php

```
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baldinof\RoadRunnerBundle\Temporal;
+
+use Temporal\Worker\WorkerOptions;
+
+final class WorkerOptionsFactory
+{
+    public static function createFromArray(array $options): WorkerOptions
+    {
+        $workerOptions = new WorkerOptions();
+
+        if (\array_key_exists('max_concurrent_activity_execution_size', $options)) {
+            /** @phpstan-ignore-next-line */
+            $workerOptions = $workerOptions->withMaxConcurrentActivityExecutionSize((int) $options['max_concurrent_activity_execution_size']);
+        }
+
+        if (\array_key_exists('worker_activities_per_second', $options)) {
+            $workerOptions = $workerOptions->withWorkerActivitiesPerSecond((float) $options['worker_activities_per_second']);
+        }
+
+        if (\array_key_exists('max_concurrent_local_activity_execution_size', $options)) {
+            /** @phpstan-ignore-next-line */
```

why'd you not specify the correct types in `@param` phpdoc instead of suppressing the errors?

you can use it like this: `/** @param array{token: string, refreshToken: string} $foo */`